blog-meeting-minutes/2024-03-08-office-hour.md

```diff
@@ -0,0 +1,44 @@
+---
+slug: office-hour-2024-03-08
+title: Office Hour 08.03.2024
+authors: 
+    - almadi_gabor
+tags: [meeting-minutes, community]
```

Comment:
```suggestion
tags: [tractus-x-office-hour,meeting-minutes, community]
```
since @hzierer introduced a new tag for our "Office Hour" we should follow the tags as well.
FYI: https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/744/files

Reply:
Thanks, missed that.